sql/executor.go

```diff
@@ -508,49 +475,10 @@ func (e *Executor) ExecuteStatements(
 		execCtx:       &e.ctx,
 	}
 
-	curTxnState := txnState{
-		txn:   nil,
-		state: session.Txn.State,
-		tr:    session.Trace,
-	}
-	txnProto := session.Txn.Txn
-	if txnProto != nil {
-		curTxnState.txn = e.newTxn(session)
-		curTxnState.txn.Proto = *txnProto
-		curTxnState.txn.UserPriority = session.Txn.UserPriority
-		curTxnState.retryIntent = session.Txn.retryIntent
-		if session.Txn.MutatesSystemConfig {
```

Comment:
`MutatesSystemConfig` only served to call `SetSystemConfigTrigger()` on the `client.Txn`s that we kept creating. Now we don't create those over and over again for every ping-pong.